api-contract/build.gradle

```diff
@@ -1,5 +1,18 @@
 
 description = ''
+configurations.all {
```

Comment:
Looks like this is a generic configuration required for all the parent & child modules, if we keep this in parent(ccpay-payment-app) only, will this not work for the child modules(/api, /api-contract, /model etc) as well ?

Reply:
No @AyanDuttaHMCTS , that doesnt work. It works only when configuration is added in all build.gradle files of child modules.

Reply:
As discussed & shown please keep the config in Root Build.gradle [all projects section]